wakuv2/waku.go

```diff
@@ -1176,6 +1182,7 @@ func (w *Waku) Start() error {
 					if err != nil {
 						w.logger.Error("OnNewEnvelopes error", zap.Error(err))
 					}
+					w.logger.Info("Got a missing message!")
```

Comment:
I think this is `Debug` or `Warning` level?

Reply:
I did not mean to add this, probably a mistake when rebasing. Removed.